spec/interactors/content/update_interactor_spec.rb

```diff
@@ -3,57 +3,63 @@
 RSpec.describe Content::UpdateInteractor do
   describe ".call" do
     before { stub_any_publishing_api_put_content }
-    let(:edition) { create(:edition) }
+    let(:edition) { create(:edition, number: 2) }
     let(:user) { build(:user) }
 
-    def build_params(document: edition.document,
-                     title: SecureRandom.alphanumeric(10),
-                     summary: SecureRandom.alphanumeric(10))
-      ActionController::Parameters.new(
-        document: document.to_param,
-        revision: { title: title, summary: summary },
-      )
+    let(:params) do
+      ActionController::Parameters.new(document: edition.document.to_param,
+                                       title: "New title",
+                                       summary: "New summary",
+                                       change_note: "New note")
```

Comment:
Should this also have update type?

Minor but I'm not sure if it's a bit of a readability detriment having this function switch to a let and having to do mutating of the variable.

Reply:
I did, briefly. Eventually I decided to take it out, since RSpec makes it hard to do a `to_not change [...].and_not change [...]` for the `change_note` and the `update_type`. I thought just checking the change note would suffice.

With the param building, I agree it's a minor point. I made the change due to having to modify the parameters and the definition (two things) of the former `build_params` method, and the slight surprise of having a helper method in a unit spec. Perhaps I'm being oversensitive to the presence of a method. Are you content to leave it as-is?

Reply:
Cool no worries about the update_type

Yeah I don't mind leaving this in. Though I very much think we should accept and welcome methods in unit tests though, after all they're the same situation as importing a helper and are often more understandable than the use of lets.